Userland/Libraries/LibJS/Runtime/Object.h

```diff
@@ -172,6 +172,8 @@ class Object : public Cell {
     bool put_own_property(const StringOrSymbol& property_name, Value, PropertyAttributes attributes, PutOwnPropertyMode = PutOwnPropertyMode::Put, bool throw_exceptions = true);
     bool put_own_property_by_index(u32 property_index, Value, PropertyAttributes attributes, PutOwnPropertyMode = PutOwnPropertyMode::Put, bool throw_exceptions = true);
 
+    friend Array;
```

Comment:
Is this ok? I needed put_own_property but did not want to duplicate the logic.
Now that I'm thinking about it maybe `define_property` just works...

Reply:
`define_property` works will push shortly